cms/templates/cms/home_page.html

```diff
@@ -68,147 +74,89 @@
         {% endif %}
         <div class="banner-wrapper-content">
           <h1>
-            MITx MicroMasters Programs: Bringing MIT to you
+            MITx MicroMasters Programs: Master Your Future
           </h1>
-          <h2>
-            Advance your career. Accelerate your master’s degree.
-          </h2>
+          <a id="actionButton" class="btn btn-primary text-uppercase px-5 py-2 action-button" data-fancybox href="https://www.youtube.com/embed/7lVvOSJpFGY?autoplay=true">Watch Now</a>
+          <p class="video-container"></p>
         </div>
       </div>
 
-      <div class="row sub-banner">
-        <div class="col-md-12 ">
+      <div class="row sub-banner current-programs-section">
+        <div class="col-md-12">
           <section class="sub-banner-section">
             <div class="sub-banner-text">
-              <h3>What is the MicroMasters<span class="trademark">&reg;</span> Program Credential?</h3>
-            <div class="youtubevideowrap">
-              <div class="video-container">
-                <iframe width="560" height="315" src="https://www.youtube.com/embed/7lVvOSJpFGY" frameborder="0" allowfullscreen></iframe>
-              </div>
-            </div>
-              <p>A MicroMasters program credential is a professional and academic credential for online learners from
-                anywhere in the world. To earn the credential, learners must pass an integrated set of MIT graduate-level courses on edX.org,
-                and one or more proctored exams. This credential is valuable in and of itself.  Additionally,
-                credential-holders can then apply for an accelerated, on campus, master’s degree program at MIT
-                or other universities.
-                {% if benefits_url %}
-                  All MicroMasters program credential holders are also considered
-                  <a href="{{benefits_url}}">affiliates of the MIT Alumni Association</a>.
-                {% endif %}
+              <h2>MITx MicroMasters<span class="trademark">&reg;</span> Programs</h2>
```

Comment:
Increase the top and bottom margin for this text. See https://invis.io/YFZ4N7KMWT3#/435513872_Homepage_V2